content/sensu-go/5.16/_index.md

```diff
@@ -27,70 +29,70 @@ Learn about [support packages](https://sensu.io/support) and [commercial feature
 
-- **Better incident response with filterable, context-rich alerts**
+### Better incident response with filterable, context-rich alerts
 
 Get meaningful alerts when and where you need them.
 Use [event filters][18] to reduce noise and [check hooks][19] to add context and speed up incident response.
 Sensu integrates with the tools and services your organization already uses like [PagerDuty][31], [Slack][29], and more.
-Check out [Bonsai, the Sensu asset index][30], or write your own [Sensu Plugins][9] in any language.
+Check out [Bonsai, the Sensu asset index][30], or write your own [Sensu plugins][9] in any language.
 
-- **Collect and store metrics with built-in support for industry-standard tools**
+### Collect and store metrics with built-in support for industry-standard tools
 
 Know what's going on everywhere in your system.
-Sensu supports industry-standard [metric formats][20] like Nagios Performance Data, Graphite Plaintext Protocol, InfluxDB Line Protocol, OpenTSDB Data Specification, and [StatsD metrics][24].
+Sensu supports industry-standard [metric formats][20] like Nagios performance data, Graphite plaintext protocol, InfluxDB line protocol, OpenTSDB data specification, and [StatsD metrics][24].
 Use the Sensu agent to collect metrics alongside check results, then use the event pipeline to route the data to a time series database like [InfluxDB][6].
 
-- **Intuitive API and dashboard interfaces**
+### Intuitive API and dashboard interfaces
 
 Sensu includes a [dashboard][25] to provide a unified view of your entities, checks, and events, as well as a user-friendly silencing tool.
 The [Sensu API][23] and the [`sensuctl` command-line tool][26] allow you (and your internal customers) to create checks, register entities, manage configuration, and more.
 
-- **Open core software backed by Sensu Inc.**
+### Open core software backed by Sensu Inc.
 
-Sensu Go's core is open source software, freely available under a
-permissive [MIT License][12] and publicly available on [GitHub][13].
+Sensu Go's core is open source software, freely available under a permissive [MIT License][12] and publicly available on [GitHub][13].
 Learn about [support packages](https://sensu.io/support) and [commercial features designed for monitoring at scale][28].
 
-[1]: getting-started/get-started
-[2]: http://slack.sensu.io/
-[3]: reference/sensuctl
-[4]: guides/aggregate-metrics-statsd
-[5]: reference/rbac
-[6]: https://influxdata.com
-[7]: https://slack.com
-[8]: https://github.com/sensu-plugins
-[9]: /plugins/latest/reference/
-[12]: https://github.com/sensu/sensu-go/blob/master/LICENSE
-[13]: https://github.com/sensu/sensu-go
+[1]: getting-started/get-started/
+[2]: https://slack.sensu.io/
+[3]: reference/sensuctl/
+[4]: guides/aggregate-metrics-statsd/
+[5]: reference/rbac/
+[6]: https://www.influxdata.com/
+[7]: https://www.slack.com/
```

Comment:
I can't find references to either of these links for this page. Sorta stumbled on it as I was randomly testing links and tried the Slack one on line 53, then that took me to the Slack handler's Bonsai page.

So not sure if things should be linking to other products pages or to bonsai near line 53.

Reply:
Thank you! I corrected the links.